basic-steps/src/main/java/org/jenkinsci/plugins/workflow/steps/ToolStep.java

```diff
@@ -80,7 +80,7 @@ public DescriptorImpl() {
         }
 
         @Override public String getDisplayName() {
-            return "Install a tool";
+            return "Use a tool";
```

Comment:
I write something like `Use a tool from a predefined Tool Installation`. It should be more explicit for both users and Jenkins maintainers